tests/rados/test_osd_db_wal_configuration.py

```diff
@@ -0,0 +1,207 @@
+                log.info(
+                    f"Creating volume group in {osd_node.hostname} osd node at {path}"
+                )
+                vg_out, _ = osd_node.exec_command(
+                    sudo=True, cmd=cmd_vgcreate, check_ec=False
+                )
+                log.info(f"The volume creation output is {vg_out}")
+                if "successfully created" not in vg_out:
+                    log.info(
+                        f"Volume group is not created in the {osd_node.hostname}.Skipping the further tests."
+                    )
+                    continue
+                log.info(
+                    f"Volume group is created at {osd_node.hostname} osd node at {path}"
+                )
+                # Create logical volume
+                log.info("Creating logical volumes")
+                cmd_lvcreate_base = "lvcreate -l 5 -n"
+                if "nvme" in path:
+                    cmd_lvcreate = f"{cmd_lvcreate_base} new_wal LVMTest -y"
+                else:
+                    cmd_lvcreate = f"{cmd_lvcreate_base} new_db LVMTest -y"
+                lv_out, _ = osd_node.exec_command(
+                    sudo=True, cmd=cmd_lvcreate, check_ec=False
+                )
+                log.info(f"Logical volumes are created in the {osd_node.hostname}")
+                if "created" not in lv_out:
+                    log.info(
+                        f"Logical volume  is not created in the {osd_node.hostname}.Skipping the further tests."
+                    )
+                    continue
+                # If in all nodes VG or lv creation fails then test_count is 0 and making the tes case fail.
+                test_count += 1
+                osds_id_in_node = rados_object.collect_osd_daemon_ids(osd_node)
+                test_osd = random.choice(osds_id_in_node)
+                log.info(f"Performing the configurations on {test_osd}")
+                # Get the OSD_ID
+                osd_fsid = rados_object.get_osd_fsid(test_osd)
+                log.info(f" The osd-{test_osd} fsid is - {osd_fsid}")
+                # stop test osd-
+                if not rados_object.change_osd_state(action="stop", target=test_osd):
+                    log.error(f"Unable to stop the OSD : {test_osd}")
+                    raise Exception("Execution error")
+                # Get the LVM list before the configuration
+                before_configure_lvm = rados_object.get_ceph_volume_lvm_list(
+                    osd_node, test_osd
+                )
+                log.info(
+                    f" ceph volume lvm list for the osd{test_osd} before configuration is {before_configure_lvm}"
+                )
+                cmd_base = f"cephadm shell --name osd.{test_osd} --"
+                if "nvme" in path:
+                    device_flag = "wal"
+                    cmd_add_db_wal = (
+                        f"{cmd_base} ceph-volume lvm new-wal --osd-id {test_osd} --osd-fsid {osd_fsid} "
```

Comment:
Please plan to add new devices via ceph-bluestore tool as well. Please create a backlog task for the same.

Reply:
Added the task to the backlog.
Jira: https://issues.redhat.com/browse/RHCEPHQE-12009